### PR TITLE
Add U2F Factors API content to developer doc

### PR DESCRIPTION
**Pull request by kentonsmith-okta**
## Description:
- Add U2F Factors API content to developer doc
- Remove EA label

### Resolves:
* [Issue #X](#X)

* [OKTA-172280](https://oktainc.atlassian.net/browse/OKTA-172280)

### CODE REVIEW

**Comment by jakubvul** [comment]
It'd be good to have some context for this - is it part of a weekly release? Seems like we're taking something out of EA?

File: _source/_docs/api/resources/factors.md [hunk 1]
```diff
@@ -1498,6 +1499,75 @@ curl -v -X POST \
   }' "https://{yourOktaDomain}.com/api/v1/users/00u15s1KDETTQMQYABRL/factors?activate=true"
 ~~~
 
+#### Enroll U2F Factor
+{:.api .api-operation}
+
+Enrolls a user with a U2F factor. The enrollment process starts with getting  nonce from Okta and using that to get registration information from the U2F key using the U2F javascript API.
```
Comment (jakubvul) [inline _source/_docs/api/resources/factors.md]: - _a_ nonce
- Javascript

File: _source/_docs/api/resources/factors.md [hunk 2]
```diff
@@ -2696,6 +2865,170 @@ curl -v -X POST \
...
+### Verify U2F Factor
+{:.api .api-operation}
+
+<span class="api-uri-template api-uri-post"><span class="api-label">POST</span> /api/v1/users/${userId}/factors/${factorId}/verify</span>
+
+Note: 
+
+According to
+[FIDO
+spec](https://fidoalliance.org/specs/fido-u2f-v1.2-ps-20170411/fido-appid-and-facets-v1.2-ps-20170411.html#h2_the-appid-and-facetid-assertions), activating and verifying a U2F device with appIds in different DNS zone is not allowed. For example, if a user activated a U2F device via the Factors API from a server hosted at https://foo.example.com, the user can verify the U2F factor from https://foo.example.com, but will not be able to verify it from Okta portal https://company.okta.com.  Here, the U2F device would return error code 4 - DEVICE_INELIGIBLE.
```
Comment (jakubvul) [inline _source/_docs/api/resources/factors.md]: - Any non-clickable URLs (ex. `https://foo.example.com`) and values (`DEVICE_INELIGIBLE`) should be code formatted.

File: _source/_docs/api/resources/factors.md [hunk 3]
```diff
@@ -2024,6 +2096,103 @@ curl -v -X POST \
...
+      var registrationData = data.registrationData;
+
+      // Get the clientData from the callback result
+      var clientData = data.clientData;
+    }
+  });
+</script>
+~~~
+
+Activate a `u2f` factor by verifying the registration data and client data.
```
Comment (jakubvul) [inline _source/_docs/api/resources/factors.md]: Any reason why it's `u2f` here but plain "U2F" everywhere else?

File: _source/_docs/api/resources/factors.md [hunk 4]
```diff
@@ -2696,6 +2865,170 @@ curl -v -X POST \
...
+      "hints":{  
+        "allow":[  
+          "GET",
+          "DELETE"
+        ]
+      }
+    }
+  },
+  "_embedded":{  
+    "enrolledFactors":[
```
Comment (srinivasanagandla-okta) [inline _source/_docs/api/resources/factors.md]: Can we exclude the `enrolledFactors` from this example as this will be gone?

**Pull request merged**